eng/common/spelling/Invoke-Cspell.ps1

```diff
@@ -167,9 +167,9 @@ $originalLocation = Get-Location
 
 try {
   Set-Location $PackageInstallCache
-  npm install npx | Out-Null
-  npm install cspell | Out-Null
-  npm install | Out-Null
+  npm ci npx | Out-Null
+  npm ci cspell | Out-Null
```

Comment:
Why do we need to specifically install npx and cspell here?

Reply:
I would also remove the `| Out-Null`, I think it's better to keep the logs than send them to null.

Reply:
This script produces output which is evaluated by other code. To that end, one should run `npm ci | Write-Host` to preserve the logs but not have the additional stdout from `npm ci` interfering with upstream callers. 

Yes, I agree we should keep the logs.